crates/sui/tests/protocol_version_tests.rs

```diff
@@ -10,9 +10,13 @@ use test_utils::authority::start_node;
 #[should_panic]
 async fn test_validator_panics_on_unsupported_protocol_version() {
     let dir = tempfile::TempDir::new().unwrap();
+    let latest_version = ProtocolVersion::MAX;
```

Comment:
this is fine, but this is actually causing a panic for a different reason, so this test is broken anyway. i need to fix it. but go ahead with this for now.